week-2/L-array-methods-2/exercise2.js

```diff
@@ -7,15 +7,22 @@
   Tip: use the string method .split() and the array method .join()
 */
 
-function capitalise(str) {}
+function capitalise(str) {
+  ["daniel"].split();
+  return "Daniel";
+}
+function capitalise() {
+  ["h", "e", "l", "l", "o"].join();
+  return "Hello";
+}
 
 /* 
   DO NOT EDIT BELOW THIS LINE
   --------------------------- */
 var name = "daniel";
 
-console.log(capitalise(name));
```

Comment:
Read the instructions on line 13(20 after your changes) `DO NOT EDIT BELOW THIS LINE`